### ISSUE TITLE
Readme enhancement 

### ISSUE THREAD

**Issue by mohitjaisal** [mohitjaisal/NASA-APOD#2]
- Improvements in readme documentation
- Demo links and Some images required
**Note:** _You can fix and improve our readme UI_

**Comment by sharonibose** [comment]
@mohitjaisal, could you kindly assign me this task?

**Comment by mohitjaisal** [comment]
Go ahead @sharonibose 👍🏻

**Comment by sharonibose** [comment]
Thank you for assigning me. I will make the necessary changes soon.

**Comment by mohitjaisal** [comment]
are you working on this @sharonibose ???

**Comment by Darius28** [comment]
hey! please assign this to me.

**Comment by mohitjaisal** [comment]
Go ahead @Darius28 👍🏻

**Comment by srai0109** [comment]
Hey @mohitjaisal , can I also work on this issue?

**Comment by mohitjaisal** [comment]
Yeah, @srai0109 👍🏻  you can also continue with @Darius28 & @sharonibose

**Comment by srai0109** [comment]
@mohitjaisal  Kindly review my PR #6 .
